README.md

```diff
@@ -18,8 +18,11 @@ pip install amdirt
 
 ### 2. With conda
 
+Installing AMDirT in a dedicated conda environment
+
 ```bash
-conda install -c bioconda amdirt
+conda create -n amdirt -c bioconda amdirt
+conda activate amdirt
 ```
 
 ### The latest development version, directly from GitHub
```

Comment:
Maybe mention for hardcore users who want to install everything manually (masochists) they can see `setup.py` for depedencies

Reply:
Or just list the dependencies

Reply:
That's a bad idea, and creates a potential issue to keep the doc and the code in sync

Reply:
Yes agreed, after reading the comment of the reviewer again I forgot to delete this comment! Sorry!